contrib/opentimelineio_contrib/adapters/tests/test_rvsession.py

```diff
@@ -468,14 +468,20 @@
     "RV Adapter does not work in python 3."
 )
 class RVSessionAdapterReadTest(unittest.TestCase):
+    def setUp(self):
+        self.fd, self.tmp_path = tempfile.mkstemp(suffix=".rv", text=True)
```

Comment:
I think you should be closing the fd here to avoid the issue of re-opening an already open file in windows.